module-gui/gui/core/FontManager.cpp

```diff
@@ -7,16 +7,37 @@
 #include "RawFont.hpp"  // for RawFont
 #include <log/log.hpp>  // for LOG_ERROR, LOG_INFO, LOG_WARN
 #include <Utils.hpp>
+#include <json11.hpp>
 #include <filesystem>
+#include <fstream>
 #include <cstdio>
 
-namespace style::window::font
-{
-    inline constexpr auto default_fallback_font = "dejavu_sans_bold_27";
-}
-
 namespace gui
 {
+    namespace fonts_map
```

Comment:
so enum was not needed in the first place? :)

Reply:
removed